src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block{
+	private String value;
```

Comment:
The name of the argument for all of your classes may not be very intuitive. What is value?